microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -0,0 +1,32 @@
+---
+description: Versioning Models used for Microsoft Edge WebView2
+title: Versioning of Microsoft Edge WebView2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/19/2020
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+
+# Browser version needed for WebView2  
+
+WebView2 is dependent upon Microsoft Edge to function.  Each WebView2 SDK has a minimum browser version required to be installed.  This browser version is specified in our [Release Notes](../releasenotes.md).  You may see the minimum version reflected in the patch version of the SDK packaged version string \(for example, `SDK package version 0.9.488` indicates a minimum browser build number of 488\).  
+
+To utilize WebView2 in your app, the minimum version of [Microsoft Edge (Chromium)](/microsoft-edge/index) supported by the WebView2 SDK must be installed on your machine.  To access the latest releases of the browser please see [Browser Channels](deployedge/microsoft-edge-channels).  
```

Comment:
The browser channels link is broken. Can you review please?

Reply:
I'm not sure the correct way to do this. I swapped back to the full link. @jm-trd-ms Had edited it to be a relative link, but maybe that isn't working. I'm not sure the full details. The full link should work.